Look like another one hidden in the logs in `execnet`
```
module>
  File "<string>", line 8, in <module>
  File "/__w/core/core/venv/lib/python3.9/site-packages/execnet/gateway_base.py", line 1554, in serve
    WorkerGateway(io=io, id=id, _startcount=2).serve()
  File "/__w/core/core/venv/lib/python3.9/site-packages/execnet/gateway_base.py", line 1060, in serve
    self._execpool.integrate_as_primary_thread()
  File "/__w/core/core/venv/lib/python3.9/site-packages/execnet/gateway_base.py", line 267, in integrate_as_primary_thread
    self._perform_spawn(reply)
  File "/__w/core/core/venv/lib/python3.9/site-packages/execnet/gateway_base.py", line 285, in _perform_spawn
    reply.run()
  File "/__w/core/core/venv/lib/python3.9/site-packages/execnet/gateway_base.py", line 220, in run
    self._result = func(*args, **kwargs)
  File "/__w/core/core/venv/lib/python3.9/site-packages/execnet/gateway_base.py", line 1084, in executetask
    do_exec(co, loc)  # noqa
  File "/__w/core/core/venv/lib/python3.9/site-packages/xdist/remote.py", line 266, in <module>
    config.hook.pytest_cmdline_main(config=config)
  File "/__w/core/core/venv/lib/python3.9/site-packages/pluggy/_hooks.py", line 265, in __call__
    return self._hookexec(self.name, self.get_hookimpls(), kwargs, firstresult)
  File "/__w/core/core/venv/lib/python3.9/site-packages/pluggy/_manager.py", line 80, in _hookexec
    return self._inner_hookexec(hook_name, methods, kwargs, firstresult)
  File "/__w/core/core/venv/lib/python3.9/site-packages/pluggy/_callers.py", line 39, in _multicall
    res = hook_impl.function(*args)
  File "/__w/core/core/venv/lib/python3.9/site-packages/_pytest/main.py", line 316, in pytest_cmdline_main
    return wrap_session(config, _main)
  File "/__w/core/core/venv/lib/python3.9/site-packages/_pytest/main.py", line 269, in wrap_session
    session.exitstatus = doit(config, session) or 0
  File "/__w/core/core/venv/lib/python3.9/site-packages/_pytest/main.py", line 323, in _main
    config.hook.pytest_runtestloop(session=session)
  File "/__w/core/core/venv/lib/python3.9/site-packages/pluggy/_hooks.py", line 265, in __call__
    return self._hookexec(self.name, self.get_hookimpls(), kwargs, firstresult)
  File "/__w/core/core/venv/lib/python3.9/site-packages/pluggy/_manager.py", line 80, in _hookexec
    return self._inner_hookexec(hook_name, methods, kwargs, firstresult)
  File "/__w/core/core/venv/lib/python3.9/site-packages/pluggy/_callers.py", line 39, in _multicall
    res = hook_impl.function(*args)
  File "/__w/core/core/venv/lib/python3.9/site-packages/xdist/remote.py", line 87, in pytest_runtestloop
    self.run_one_test(torun)
  File "/__w/core/core/venv/lib/python3.9/site-packages/xdist/remote.py", line 106, in run_one_test
    self.config.hook.pytest_runtest_protocol(item=item, nextitem=nextitem)
  File "/__w/core/core/venv/lib/python3.9/site-packages/pluggy/_hooks.py", line 265, in __call__
    return self._hookexec(self.name, self.get_hookimpls(), kwargs, firstresult)
  File "/__w/core/core/venv/lib/python3.9/site-packages/pluggy/_manager.py", line 80, in _hookexec
    return self._inner_hookexec(hook_name, methods, kwargs, firstresult)
  File "/__w/core/core/venv/lib/python3.9/site-packages/pluggy/_callers.py", line 39, in _multicall
    res = hook_impl.function(*args)
  File "/__w/core/core/venv/lib/python3.9/site-packages/_pytest/runner.py", line 109, in pytest_runtest_protocol
    runtestprotocol(item, nextitem=nextitem)
  File "/__w/core/core/venv/lib/python3.9/site-packages/_pytest/runner.py", line 126, in runtestprotocol
    reports.append(call_and_report(item, "call", log))
  File "/__w/core/core/venv/lib/python3.9/site-packages/_pytest/runner.py", line 215, in call_and_report
    call = call_runtest_hook(item, when, **kwds)
  File "/__w/core/core/venv/lib/python3.9/site-packages/_pytest/runner.py", line 254, in call_runtest_hook
    return CallInfo.from_call(
  File "/__w/core/core/venv/lib/python3.9/site-packages/_pytest/runner.py", line 311, in from_call
    result: Optional[TResult] = func()
  File "/__w/core/core/venv/lib/python3.9/site-packages/_pytest/runner.py", line 255, in <lambda>
    lambda: ihook(item=item, **kwds), when=when, reraise=reraise
  File "/__w/core/core/venv/lib/python3.9/site-packages/pluggy/_hooks.py", line 265, in __call__
    return self._hookexec(self.name, self.get_hookimpls(), kwargs, firstresult)
  File "/__w/core/core/venv/lib/python3.9/site-packages/pluggy/_manager.py", line 80, in _hookexec
    return self._inner_hookexec(hook_name, methods, kwargs, firstresult)
  File "/__w/core/core/venv/lib/python3.9/site-packages/pluggy/_callers.py", line 39, in _multicall
    res = hook_impl.function(*args)
  File "/__w/core/core/venv/lib/python3.9/site-packages/_pytest/runner.py", line 162, in pytest_runtest_call
    item.runtest()
  File "/__w/core/core/venv/lib/python3.9/site-packages/_pytest/python.py", line 1641, in runtest
    self.ihook.pytest_pyfunc_call(pyfuncitem=self)
  File "/__w/core/core/venv/lib/python3.9/site-packages/pluggy/_hooks.py", line 265, in __call__
    return self._hookexec(self.name, self.get_hookimpls(), kwargs, firstresult)
  File "/__w/core/core/venv/lib/python3.9/site-packages/pluggy/_manager.py", line 80, in _hookexec
    return self._inner_hookexec(hook_name, methods, kwargs, firstresult)
  File "/__w/core/core/venv/lib/python3.9/site-packages/pluggy/_callers.py", line 39, in _multicall
    res = hook_impl.function(*args)
  File "/__w/core/core/venv/lib/python3.9/site-packages/aiohttp/pytest_plugin.py", line 186, in pytest_pyfunc_call
    _loop.run_until_complete(pyfuncitem.obj(**testargs))
  File "/usr/local/lib/python3.9/asyncio/base_events.py", line 629, in run_until_complete
    self.run_forever()
  File "/usr/local/lib/python3.9/asyncio/base_events.py", line 596, in run_forever
    self._run_once()
  File "/usr/local/lib/python3.9/asyncio/base_events.py", line 1882, in _run_once
    handle._run()
  File "/usr/local/lib/python3.9/asyncio/events.py", line 80, in _run
    self._context.run(self._callback, *self._args)
  File "/__w/core/core/tests/conftest.py", line 686, in point_in_time_listener
    hass.async_run_hass_job(job, now)
  File "/__w/core/core/homeassistant/core.py", line 436, in async_run_hass_job
    return self.async_add_hass_job(hassjob, *args)
  File "/__w/core/core/homeassistant/core.py", line 368, in async_add_hass_job
    task = self.loop.run_in_executor(  # type: ignore
  File "/usr/local/lib/python3.9/asyncio/base_events.py", line 813, in run_in_executor
    return futures.wrap_future(
  File "/usr/local/lib/python3.9/asyncio/futures.py", line 412, in wrap_future
    new_future = loop.create_future()
  File "/usr/local/lib/python3.9/asyncio/base_events.py", line 424, in create_future
    return futures.Future(loop=self)
Traceback (most recent call last):
  File "/usr/local/lib/python3.9/concurrent/futures/thread.py", line 52, in run
    result = self.fn(*self.args, **self.kwargs)
  File "/__w/core/core/homeassistant/components/rflink/binary_sensor.py", line 88, in off_delay_listener
    self.async_write_ha_state()
  File "/__w/core/core/homeassistant/helpers/entity.py", line 506, in async_write_ha_state
    self._async_write_ha_state()
  File "/__w/core/core/homeassistant/helpers/entity.py", line 630, in _async_write_ha_state
    self.hass.states.async_set(
  File "/__w/core/core/homeassistant/core.py", line 1232, in async_set
    self._bus.async_fire(
  File "/__w/core/core/homeassistant/core.py", line 734, in async_fire
    self._hass.async_add_hass_job(job, event)
  File "/__w/core/core/homeassistant/core.py", line 365, in async_add_hass_job
    self.loop.call_soon(hassjob.target, *args)
  File "/usr/local/lib/python3.9/asyncio/base_events.py", line 748, in call_soon
    self._check_thread()
  File "/usr/local/lib/python3.9/asyncio/base_events.py", line 785, in _check_thread
    raise RuntimeError(
RuntimeError: Non-thread-safe operation invoked on an event loop other than the current one
```

--- a/.github/workflows/ci.yaml
+++ b/.github/workflows/ci.yaml
@@ -14,6 +14,7 @@ env:
   DEFAULT_PYTHON: 3.8
   PRE_COMMIT_CACHE: ~/.cache/pre-commit
   SQLALCHEMY_WARN_20: 1
+  PYTHONASYNCIODEBUG: 1
 
 concurrency:
   group: ${{ github.workflow }}-${{ github.event.pull_request.number || github.ref }}

